docs/parameters.md

```diff
@@ -2,34 +2,35 @@
-| NUM_PROC_COLS       | int   | Number of processors to use for the columns                         | 1                                          |
-| NUM_PROC_ROW        | int   | Number of processors to use for the rows                            | 1                                          |
-| MAX_ITERATIONS      | int   | Maximum successful iterations performed by fourdvar                 | 20                                         |
+| Variable           | Type  | Description                                                         | Default                                    |
+|--------------------|-------|---------------------------------------------------------------------|--------------------------------------------|
+| DOMAIN_NAME        | str   | Defines the target domain                                           | aust10km                                   |
+| DOMAIN_VERSION     | str   | Version of the target domain                                        | v1                                         |
+| DOMAIN_MCIP_SUFFIX | str   | Suffix for the generated MCIP files                                 | ${DOMAIN_NAME}_${DOMAIN_VERSION}           |
+| START_DATE         | date  | Start date of the run                                               | 2022-07-01                                 |
+| END_DATE           | date  | End date of the run (inclusive)                                     | 2022-07-30                                 |
+| STORE_PATH         | str   | Full path to the branch-specific data.                              | N/A                                        |          
+| EXPERIMENT         | str   | Name of the experiment being run                                    | 202207_test                                |
+| TEMPLATE_DIR       | str   | Path to the CMAQ template directory                                 | {STORE_PATH}/templates                    |
+| CMAQ_SOURCE_DIR    | str   | Path to the root of the CMAQ source directory                       | N/A                                        |
+| MCIP_SOURCE_DIR    | str   | Path to the root MCIP source directory                              | N/A                                        |
+| MET_DIR            | path  | Output directory for the MCIP data                                  | N/A                                        |
+| CTM_DIR            | path  | Output directory for the CMAQ template files                        | N/A                                        |
+| WRF_DIR            | path  | Output directory for the WRF outputs (from setup-wrf)               | N/A                                        |
+| GEO_DIR            | path  | Directory containing the `geo_em.d??.nc` file (from setup-wrf)      | N/A                                        |
+| CHK_PATH            | path  | Directory to store CMAQ checkpoint files                            | {CMAQ_BASE}/chkpnt                         |
```

Comment:
And added this.

The naming is a bit awkward here. CHK_PATH is the name used by CMAQ, but CHK_DIR would be more consistent...